clientlibrary/worker/worker.go

```diff
@@ -324,6 +375,90 @@ func (w *Worker) eventLoop() {
 	}
 }
 
+func (w *Worker) rebalance() error {
+	log := w.kclConfig.Logger
+
+	workers, err := w.leasestealer.ListActiveWorkers(w.shardStatus)
```

Comment:
Those are workers with at least 1 shard. There might be some workers don't have shard at all (idle).

Reply:
I'm not sure what you mean here. Could you elaborate a bit more?